corehq/apps/reports/tasks.py

```diff
@@ -294,31 +294,30 @@ def _format_filename(form_info, question_id, extension, case_id_to_name):
 def _write_attachments_to_file(fpath, num_forms, forms_info, case_id_to_name):
     total_size = 0
     unique_attachment_ids = set()
-    with open(fpath, 'wb') as zfile:
-        with zipfile.ZipFile(zfile, 'w') as multimedia_zipfile:
```

Comment:
Huh, so I'm reading this as, ZipFile will accept either a file object or a file path (string), but also that if you pass it both a file object _and_ a `mode` (`'w'` here), it will... I guess just ignore the `mode` argument?

Anyway looks like good cleanup, but odd behavior from `zipfile`!